common-custom-user-data-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java-gradle-plugin'
+    id 'maven-publish'
+}
+
+group = "com.gradle"
+version = "0.1"
+
+repositories {
```

Comment:
For the group and artifact, I suggest to use what we use for Maven but exchange 'Gradle':

```
<dependency>
  <groupId>com.gradle</groupId>
  <artifactId>common-custom-user-data-gradle-plugin</artifactId>
  <version>1.0</version>
</dependency>
```

Reply:
This is already the case, except for the temporary version `0.1`